students/TrkCat/Lesson02/drawGrid.py

```diff
@@ -0,0 +1,34 @@
+def gridPart1():
```

Comment:
per pep8 (python style guide): function names and var names should be snake_case -> `grid_part1()`
same for modules (file names) -> `draw_grid.py`

Reply:
Thanks! It'll take me awhile to break my camel case habits.

Reply:
understandable, it's challenging to switch between different languages with different style guidelines 🙂